recipe/patch_yaml/scipy.yaml

```diff
@@ -53,3 +53,21 @@ if:
 
 then:
   - add_constrains: libopenblas <0.3.26
+
+---
+# Align constrains with limits in 1.8.x maintenance branch
+# The numpy min and max versions are documented here:
+# https://github.com/scipy/scipy/blob/maintenance/1.8.x/setup.py
+# https://github.com/scipy/scipy/blob/maintenance/1.8.x/scipy/__init__.py
+if:
+  name: scipy
+  version: 1.8.1
+  build_number_in: [0, 1, 2]
```

Comment:
I was surprised that build 1 doesn't show up in the diff, but that's because we apparently went from build number 0 to 2 without ever publishing builds for 1.